src/cljs/athens/electron.cljs

```diff
@@ -323,6 +324,55 @@
   ;; open or create a new starter db
 
   ;; Watch filesystem, e.g. in case db is updated via Dropbox sync
+  
+  (def schemaTest
+  {:schema/version {}
+   :block/uid      {:db/unique :db.unique/identity}
+   :node/title     {:db/unique :db.unique/identity}
+   :attrs/lookup   {:db/cardinality :db.cardinality/many}
+   :block/children {:db/cardinality :db.cardinality/many
+                    :db/valueType :db.type/ref}
+   :block/refs     {:db/cardinality :db.cardinality/many
+                    :db/valueType :db.type/ref}}
+  )
+  
+  ;; Create an empty Db for test 
+  (def test-db
+    (-> (d/empty-db schemaTest))
+  )
```

Comment:
Generally not a good idea to put tests in normal code.

Reply:
Hi here I try and create a ref object that is later used to check whether the imported obj is not corrupted( correctly mapped)
This schemaTest and test-db should be name according to the naming style thanks

Reply:
you don't actually need to create `test-db` at all, I think. you only use `test-db` in `(:schema test-db)`, so you might as well just not create the `test-db` and just test `(= (:schema db) schemaTest)`

Reply:
also it would be better to reuse the schema definition from `db.cljs` instead of repeating here.

Reply:
done